Dockerfile

```diff
@@ -1,9 +1,6 @@
-# Building on Ubuntu Bionic (18.04) and supporting glibc 2.27. This allows
-# the following distros (and newer versions) to run the resulting binaries:
-# * Ubuntu Bionic (2.27)
-# * Debian Buster (2.28)
-# * CentOS 8 (2.28)
-FROM ubuntu:bionic as builder
+FROM ubuntu:20.04 as builder
+
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
It worked without this env var, do we need it for something specific with your changes?

Reply:
Yes, /usr/lib/pbuilder/pbuilder-satisfydepends will install the package tzdata, and tzdata will request to set the Geographic area and time zonem, so this env is needed.

Reply:
`pbuilder-satisfydepends` was there before and it worked. Is it related to bionic -> focal bump then?